src/webxr/ARButton.ts

```diff
@@ -79,7 +88,8 @@ class ARButton {
 
       button.onclick = function () {
         if (currentSession === null) {
-          navigator.xr.requestSession('immersive-ar', sessionInit).then(onSessionStarted)
+          const xr: THREE.XR = (navigator as any).xr
```

Comment:
unconventional, but i think i'd prefer a `@ts-expect-error` here instead, small description afterwards explaining that we need to add webxr types which *might* be an open issue?

Reply:
makes sense to me, will update